lib/scenic/adapters/postgres.rb

```diff
@@ -86,8 +99,10 @@ def drop_materialized_view(name)
       # @return [void]
       def refresh_materialized_view(name, concurrently: false)
         if concurrently
+          raise Scenic::Postgres::ConcurrentRefreshesNotSupportedError unless connection.supports_concurrent_refresh?
           execute "REFRESH MATERIALIZED VIEW CONCURRENTLY #{name};"
         else
+          raise_unless_materialized_views_supported
```

Comment:
Should this be above the conditional for `concurrently`? Seems like it applies to both branches. Granted, I can't imagine ever being in a state where this error comes up, so we're just splitting hairs here...